src/index.ts

```diff
@@ -65,27 +63,29 @@ const config = ({ domain = null, type = "localStorage" }: LocalitConfig): void =
 const set = (key: string, value: any, expirationTime?: string): void => {
   if (!key) return console.error("Localit: provide a key to store the value");
 
-  if (typeof value === "object") value = JSON.stringify(value);
+  const storeObject: LocalitStore = {
+    value,
+    meta: {
+      created: new Date().getTime(),
+      expire: expirationTime ? setExpiration(expirationTime): undefined
```

Comment:
This can also be `expire: expirationTime && setExpiration(expirationTime)`. The fallback would be a `false`, so I think that works too for `expire`.